reference/format/search.rst

```diff
@@ -1,2 +1,137 @@
 Search Index
 ============
+
++-----------+------------------------------------------------------------+
+| Filename  | ``search.yml``                                             |
```

Comment:
Please use documentation from Search bundle to do correct descriptions of this file structure https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/SearchBundle/Resources/doc/configuration.md#entity-configuration